src/Manager.js

```diff
@@ -1,27 +1,61 @@
 import User from './User'
 
 class Manager extends User {
-  constructor(date, rooms, bookings) {
-    super(id);
-    this.date = date;
+  constructor(customers, rooms, id, date, bookings) {
+    super(id, date, bookings);
+    this.customers = customers;
     this.rooms = rooms;
+    this.currentCustomer = {};
```

Comment:
I like the idea of making this property and empty object. I left my null. This looks 'prettier' to me! :)